### PR TITLE
[QA-1682] Fix automation tests after delta layer removal

### PR DESCRIPTION
**Pull request by jordansafer**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check that the **Product Owner** has signed off on any user-facing changes
- [x] **Submitter**: Make sure Swagger is updated if API changes
  - [x] **...and Orchestration's Swagger too!**
- [x] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Database checks:
  * If PR includes new or changed db queries, include the explain plans in the description
  * Make sure liquibase is updated if appropriate
  * If doing a migration, take a backup of the
  [dev](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/backups?project=broad-dsde-dev&organizationId=548622027621)
  and
  [alpha](https://console.cloud.google.com/sql/instances/terraform-r4caezzc35c4tb7pgdhwkmme4y/backups?project=broad-dsde-alpha&organizationId=548622027621)
  DBs in Google Cloud Console
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] Tell your tech lead (TL) that the PR exists if they want to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via Slack and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

Remove automation tests that depend on the delta layer
https://broadworkbench.atlassian.net/browse/QA-1682

### CODE REVIEW

File: automation/src/test/scala/org/broadinstitute/dsde/test/api/SnapshotAPISpec.scala [hunk 1]
```diff
@@ -165,84 +165,6 @@ class SnapshotAPISpec extends AnyFreeSpecLike with Matchers with BeforeAndAfterA
...
-          createSnapshotReference(billingProject, workspaceName, dataRepoSnapshotId, snapshotName)
-
-          // validate the snapshot was added correctly: list snapshots in Rawls, should return 1, which we just added.
-          val listResponse = listSnapshotReferences(billingProject, workspaceName)
-          val resources = Rawls.parseResponseAs[SnapshotListResponse](listResponse).gcpDataRepoSnapshots
-          resources should have size 1
-
-          // check that the bq dataset has been created
-          val workspaceId = resources.head.getMetadata.getWorkspaceId
-          val dataset = getDataset(generateReferenceName(workspaceId), googleProject.value, ownerAuthToken)
```
Comment (davidangb) [inline automation/src/test/scala/org/broadinstitute/dsde/test/api/SnapshotAPISpec.scala:194]: I believe the `generateReferenceName` method is now unused and can be deleted

**Review by davidangb — APPROVED** [review]
one minor tweak inline, otherwise :+1: and thanks for fixing!

If you only delete that now-unused method, I don't need to re-review.

File: automation/src/test/scala/org/broadinstitute/dsde/test/api/SnapshotAPISpec.scala (hunk 1 above)
Comment (jordansafer) [reply automation/src/test/scala/org/broadinstitute/dsde/test/api/SnapshotAPISpec.scala:194]: Yup, it is unused, fixed!

**Comment by jordansafer** [comment]
jenkins retest

**Comment by jordansafer** [comment]
jenkins retest

**Pull request merged**